app/javascript/components/visualization/ScatterPlot.jsx

```diff
@@ -598,16 +609,23 @@ function RawScatterPlot({
   return (
     <div className="plot">
       { ErrorComponent }
-      <PlotTitle
-        titleTexts={titleTexts}
-        isCorrelatedScatter={isCorrelatedScatter}
-        correlation={bulkCorrelation}/>
+      { hasMissingAnnot &&
+        <div className="alert-warning text-center error-boundary">
+          {cluster} does not have the requested annotation {loadedAnnotation}
```

Comment:
Adding quotes would help visually segment these strings, which especially helps in warning or error messages:

```suggestion
          "{cluster}" does not have the requested annotation "{loadedAnnotation}"
```